### ISSUE TITLE
Добавление бота в ТГ-чат со множеством участников. 

### ISSUE THREAD

**Issue by Kirilyaka** [nixys/nxs-support-bot#12]
Добавить возможность добавить бота не просто в ТГ клиент каждому человеку, а именно добавить бота в чат, где находится много пользователей, что бы все участники могли получить апдейт сразу. Сейчас это можно решить только тем, что человека надо выставлять "Наблюдателем" и тогда определенный круг лиц будет получать инфу, помимо того, кто выставил задачу и на кого была назначена.

**Comment by AndreiMuvila** [comment]
Здравствуйте.

На текущий момент такого функционала нет. Возможно требуется добавить режим для работы в групповых чатах.

Как именно вы видите это?

- Бот присылает уведомления в чат, но без функционала создания задач и ответа на комментарии?
- Бот прикрепляет переписку из чата в какую либо из предварительно заданных задач?

**Comment by Kirilyaka** [comment]
Нет, кажется, что бот должен просто добавляться в чат и иметь точно такой же функционал, как и сейчас. 

Сложность, я так понимаю в том, что он сейчас привязывается к ключу пользователя и от этого работает. 
А у проекта такого ключа будто бы нет, поэтому здесь видится сложность в самой реализации.
